actors/common/src/main/java/org/sunbird/common/quartz/scheduler/OnDemandShadowManager.java

```diff
@@ -0,0 +1,216 @@
+           LoggerEnum.INFO.name());
+  }
+
+  private void scheduleBulkUploadJobOnDemand(String identifier) {
+   ProjectLogger.log(
+           "OnDemandShadowManager:scheduleBulkUploadJobOnDemand:scheduler started",
+           LoggerEnum.INFO.name());
+   JobDetail uploadVerifyJob =
+           JobBuilder.newJob(UploadLookUpScheduler.class)
+                   .requestRecovery(true)
+                   .withDescription("Scheduler for bulk upload retry")
+                   .withIdentity("uploadVerifyScheduler", identifier)
+                   .build();
+   Trigger uploadTrigger =
+           TriggerBuilder.newTrigger()
+                   .withIdentity("uploadVerifyTrigger", identifier).startNow()
+                   .withSchedule(simpleSchedule().withIntervalInSeconds(1).withRepeatCount(0))
+                   .build();
+   try {
+    scheduler.scheduleJob(uploadVerifyJob, uploadTrigger);
+    scheduler.start();
+    ProjectLogger.log(
+            "OnDemandShadowManager:scheduleBulkUploadJobOnDemand: UploadLookUpScheduler schedular started",
+            LoggerEnum.INFO.name());
+   } catch (Exception e) {
+    ProjectLogger.log("OnDemandShadowManager:scheduleBulkUploadJobOnDemand Error occurred " + e.getMessage(),
+            LoggerEnum.ERROR.name());
+   }
+  }
+
+  private void scheduleShadowUserOnDemand(String identifier) {
+   ProjectLogger.log(
+           "OnDemandShadowManager:scheduleShadowUserOnDemand:scheduler started",
+           LoggerEnum.INFO.name());
+   JobDetail migrateShadowUserJob =
+           JobBuilder.newJob(ShadowUserMigrationScheduler.class)
+                   .requestRecovery(true)
+                   .withDescription("Scheduler for migrating shadow user ")
+                   .withIdentity("migrateShadowUserScheduler", identifier)
+                   .build();
+   Trigger migrateShadowUserTrigger =
+           TriggerBuilder.newTrigger()
+                   .withIdentity("migrateShadowUserTrigger", identifier).startNow()
+                   .withSchedule(simpleSchedule().withIntervalInSeconds(1).withRepeatCount(0))
+                   .build();
+   try {
+    if (scheduler.checkExists(migrateShadowUserJob.getKey())) {
+     scheduler.deleteJob(migrateShadowUserJob.getKey());
+    }
+    scheduler.scheduleJob(migrateShadowUserJob, migrateShadowUserTrigger);
+    scheduler.start();
+    ProjectLogger.log(
+            "OnDemandShadowManager:scheduleShadowUserOnDemand:scheduler ended",
+            LoggerEnum.INFO.name());
+   } catch (Exception e) {
+    ProjectLogger.log(
+            "OnDemandShadowManager:scheduleShadowUserOnDemand Error occurred " + e.getMessage(),
+            LoggerEnum.ERROR.name());
+   }
+  }
+
+  /**
+   * This method will do the Quartz scheduler set up in cluster mode.
+   *
+   * @return Properties
+   * @throws IOException
+   */
+  public Properties setUpClusterMode() throws IOException {
```

Comment:
this won't be needed , if we are going to use schedulermanager job class reference.

Reply:
reused it from existing schedulermanager class